docs/userguide.md

```diff
@@ -174,6 +175,13 @@ Status:
     Status:                True
     Type:                  Ready
 ```
+### Unjoin Clusters
+
+If required, federation allows you to unjoin clusters using `kubefed2` tool.
```

Comment:
How about clarify the what the command can help like `The following command un-joins cluster2 from federation controller`?

Reply:
Can you please elaborate.
I am trying to match the format of the whole document.
In the above command it is mentioned `--host-cluster-context`, so I feel this is understood.

Thanks

Reply:
Alright, how about use the following command instead?

```
./bin/kubefed2 unjoin cluster2 --cluster-context cluster2 --host-cluster-context cluster1 --remove-from-registry --v=2
```

Reply:
Ya, I was about to comment the same.
Thank you for your inputs.

Will update it with a new patchset

Reply:
Would love to get an opinion, whether it is required to add `--remove-from-registry` in the command

Reply:
Since the `join` is including `add-to-registry` https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md#join-clusters , so I'd prefer adding `remove-from-registry` for unjoin

Reply:
Please review @gyliu513